### PR TITLE
Chore Update

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/entity-browser/tests/test-bundler.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
+import Enzyme from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
```
Comment (rzueger) [inline packages/entity-browser/tests/test-bundler.js]: I think we should move this import to a separate file (maybe package `tocco-test-util`) and import this file in every test-bundler? I reckon we won't remain with `enzyme-adapter-react-16` once React 17 is here and we don't want to update each test-bundler.js every time React is updated.
```
import Adapter from 'enzyme-adapter-react-16'

export default Adapter
```

File: packages/tocco-ui-showcase/src/components/ShowCaseApp.spec.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```
Comment (rzueger) [inline packages/tocco-ui-showcase/src/components/ShowCaseApp.spec.js]: What's that needed for? If I remove this line and run the linter locally, it doesn't report any errors.

File: packages/tocco-ui/src/FormattedValue/typeFormatters/DocumentFormatter.spec.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```
Comment (rzueger) [inline packages/tocco-ui/src/FormattedValue/typeFormatters/DocumentFormatter.spec.js]: What's that needed for? If I remove this line and run the linter locally, it doesn't report any errors.

File: packages/tocco-ui/src/LoadMask/example.js [hunk 4]
```diff
@@ -7,15 +7,24 @@ class Example extends React.Component {
...
+    return true
+  }
+
   render() {
     return (
       <div style={{height: '100px'}}>
         {/* start example */}
-        <button onClick={() => this.setState({loaded: {}})}>Load</button>
+        <button onClick={() => {
+          console.log('this.state.loaded', this.state.loaded)
```
Comment (rzueger) [inline packages/tocco-ui/src/LoadMask/example.js]: Do we want to keep this console log in this example?

File: packages/tocco-util/src/notifier/components/Notifier.spec.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
+/* eslint-disable no-unused-vars */
```
Comment (rzueger) [inline packages/tocco-util/src/notifier/components/Notifier.spec.js]: What's that needed for? Can we instead just cleanup the imports?

Comment (MethodenMann) [reply packages/tocco-util/src/notifier/components/Notifier.spec.js]: you're right. i added these ignores for developing purposes and forgot to remove them afterwards. tanks

File: packages/tocco-test-util/src/testSetup/bundle.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import { configure } from 'enzyme'
```
Comment (rzueger) [inline packages/tocco-test-util/src/testSetup/bundle.js]: Is there no linting rule for the spaces inside `{...}` or is this file excluded from the linting? (as far as I can tell, we prefer the style without spaces)

**Pull request merged**